x/ccv/child/keeper/keeper.go

```diff
@@ -314,3 +316,16 @@ func (k Keeper) VerifyParentChain(ctx sdk.Context, channelID string) error {
 
 	return nil
 }
+
+// GetLastUnbondingPacket returns the last unbounding packet stored in lexical order
+func (k Keeper) GetLastUnbondingPacket(ctx sdk.Context) ccv.ValidatorSetChangePacketData {
+	ubdPacket := &channeltypes.Packet{}
+	k.IterateUnbondingPacket(ctx, func(seq uint64, packet channeltypes.Packet) bool {
```

Comment:
You should be able to iterate in reverse order and just return on first element. See store iteration methods